tools/codegen/README.md

```
@@ -0,0 +1,41 @@
+# Weave SDK Code Generator
+
+This tool generates code from the OpenAPI specification for Python, Node.js, and TypeScript using Stainless.
+
+## Setup
+
+1. Install weave locally:
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)